.azure-pipelines/pipelines.yml

```diff
@@ -141,6 +187,8 @@ jobs:
     strategy:
       maxParallel: 3
       matrix:
+        api:
+          CI_TARGET: "bazel.api"
```

Comment:
@mattklein123 CI job is here to be included in presubmit

Reply:
Oh I see. Sorry missed this.